modules/objc/examples/code_snippets/SampleCodeTest.m

```diff
@@ -247,13 +250,26 @@ - (void) dontTestFileLogging {
     // end::file-logging[]
 }
 
+- (void) dontTestNewFileLogging {
+    // tag::new-file-logging[]
+    NSString* tempFolder = [NSTemporaryDirectory() stringByAppendingPathComponent:  @"cbllog"];
+    CBLLogSinks.file = [[CBLFileLogSink alloc] initWithLevel:kCBLLogLevelInfo directory:tempFolder];;
```

Comment:
double `;;`

Reply:
I think it's a good idea to set he log level to verbose as that is what we normally want users to do.